### DISCUSSION TITLE
Make the documentation open source

### DISCUSSION THREAD

**Discussion by DontFred** [Dokploy/dokploy#84] [Ideas]
Maybe you could change the documentation to an open source project as well.  Then we could help you to maintain it. Possible is to use [nextra](https://nextra.site/) with markdown for example.

**Comment by Siumauricio** [comment]
I didn't read this until now apologies, in fact if that was my idea I made several transitions from mintlify which was paid to astro starlight but I didn't like it much, in the end someone in the discord recommended me fumadocs which is super good! 

https://docs.dokploy.com/en/get-started/introduction

**Reply by DontFred** [reply]
Fumadocs looks good. I have seen both and I agree astro wasn't quite fitting the dokploy thing.
